### PR TITLE
feat: Add CCameraAngles, CCameraManagerAngles, CGameCameraAngles, CPlayerAngles

### PR DESCRIPTION
**Pull request by davidebeatrici**

### CODE REVIEW

**Comment by Yimura** [comment]
Just wondering, you got any signatures or addresses for these classes?
I'd like to verify what is added actually exists in the game.

Alternatively linking to other projects that use these classes...

**Comment by davidebeatrici** [comment]
https://github.com/mumble-voip/mumble/pull/5704

https://github.com/mumble-voip/mumble/blob/414ab61b61b0d446eccba1c60dcad7c158a7c701/plugins/gtav/Game.cpp#L13-L34
https://github.com/mumble-voip/mumble/blob/414ab61b61b0d446eccba1c60dcad7c158a7c701/plugins/gtav/Game.cpp#L61-L67

The only field I'm not sure about is the `CPlayerCameraData` one.

**Comment by davidebeatrici** [comment]
I see the repository has been restructured.

Should I move the headers into their corrispettive folders (`camera`, `player`)?

**Comment by Yimura** [comment]
If you'd do that, that'd be amazing. Eventually I would've probably done it myself to not bother you with the request.

Thanks in advance!

**Comment by davidebeatrici** [comment]
No problem!

File: camera/CCameraAngles.hpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "../player/CPlayerAngles.hpp"
+
+class CCameraAngles
+{
+public:
+	char pad_0000[960]; //0x0000
+	CPlayerAngles* angles; //0x03C0
```
Comment (davidebeatrici) [inline camera/CCameraAngles.hpp:9]: Missing `m_` prefix, sorry.

Comment (Yimura) [reply camera/CCameraAngles.hpp:9]: It's fine we everything is fixable, I just was too happy to merge.

**Pull request merged**